build.sbt

```diff
@@ -3,14 +3,13 @@ import de.heikoseeberger.sbtheader.HeaderPattern
 
 name := "akka-stream-kafka"
 
-val akkaVersion = "2.4.17"
+val akkaVersion = "2.5.0"
```

Comment:
This is not possible. Akka being backwards compatible means that libraries built with Akka 2.4.x can be used in applications running Akka 2.5.x, but not the opposite.

I haven't looked at what the reason was why akka-stream-kafka could not be used with Akka 2.5.0, but I would guess we accidentally used some internal APIs in akka-stream-kafka. We should try to fix that instead, and release a new version, built with 2.4.17, that can be used with 2.5.0.